packages/prettier-plugin-java/test/unit-test/throws/_output.java

```diff
@@ -8,26 +8,65 @@ void throwException2(String string) throws RuntimeException {
     throw new RuntimeException();
   }
 
-  void throwException3(String string1, String string2, String string3)
-    throws RuntimeException {
+  void throwException3(String string1, String string2, String string3) throws
+    RuntimeException {
```

Comment:
I'm a bit worried about the invariable split of the `throws` keyword from the exception(s).
FYI I've added a very late comment to https://github.com/jhipster/prettier-java/issues/286#issuecomment-561104810